src/planscape/base/condition_types.py

```diff
@@ -21,7 +21,7 @@
 from typing import Any, Optional, TypedDict
 from typing_extensions import NotRequired
 
-Condition = npt.NDArray[np.uint16]
+Condition = npt.NDArray[np.float64]
```

Comment:
Does this work with float32?  Seems more consistent with what's below.

Reply:
Changed